src/languages/lua.coffee

```diff
@@ -19,4 +19,9 @@ module.exports = {
 
   defaultBeautifier: "Lua beautifier"
 
+  options:
+    do_not_change_whitespace:
```

Comment:
yes, I agree with that.
maybe someone like use `fn(arg1)` and other people like `fn( arg1 )`

It just give users  a choose to only format indent, but don't adjust their whitespace,

en...,what ever, I will remove this option.